version/version.go

```diff
@@ -61,6 +58,9 @@ func init() {
 			}
 		}
 	}
+	if commitHashAbbrev == "" && len(GitCommit) >= 9 {
```

Comment:
how `commitHashAbbrev` _not_ be the empty string at this point?  Remove that first part of the test? And move the decl down?

Reply:
Good catch, fixed up.